general/g.version/main.c

```diff
@@ -109,40 +114,118 @@ int main(int argc, char *argv[])
 
     shell = G_define_flag();
     shell->key = 'g';
-    shell->description =
-        _("Print info in shell script style (including Git reference commit)");
+    shell->label = _("Print info in shell script style (including Git "
+                     "reference commit) [deprecated]");
+    shell->description = _(
+        "This flag is deprecated and will be removed in a future release. Use "
+        "format=shell instead.");
     shell->guisection = _("Shell");
 
+    fopt = G_define_standard_option(G_OPT_F_FORMAT);
+    fopt->options = "plain,shell,json";
```

Comment:
This seems to solve the issue:

```suggestion
    fopt->required = NO;
    fopt->options = "plain,shell,json";
```